core/src/main/java/org/apache/accumulo/core/iterators/ColumnFamilyCounter.java

```diff
@@ -89,4 +89,11 @@ public Value getTopValue() {
     return null;
   }
 
+  @Override
```

Comment:
> Any ones in particular you had in mind

Negative from me. I just wouldn't be surprised if there were simplifications to be had.